src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding_editor.yaml

```diff
@@ -0,0 +1,12 @@
+Type: Jupyter Notebook Extension
+Name: Codefolding in Editor
+Description: |
+  Enables the CodeMirror feature to allow codefolding in the Jupyter file
+  editor view.
+  Note that this also uses the codefolding hotkey from the codefolding
+  nbextension for the notebook view.
```

Comment:
Well, it's possible, since i found that using codemirror to bind hotkeys (as I had to do in edit mode because the keyboard manager isn't in use in edit view) means that many that work in notebook view aren't recognised in the edit view. Perhaps I should try adding a keyboard manager...